report_py3o/models/py3o_report.py

```diff
@@ -244,60 +241,63 @@ def _create_single_report(self, model_instance, data, save_in_attachment):
         """ This function to generate our py3o report
         """
         self.ensure_one()
-        report_xml = self.ir_actions_report_xml_id
-        filetype = report_xml.py3o_filetype
         result_fd, result_path = tempfile.mkstemp(
-            suffix='.' + filetype, prefix='p3o.report.tmp.')
+            suffix='.ods', prefix='p3o.report.tmp.')
```

Comment:
Not sure I got it, why is .ods suffix fixed ? Single report is not also about odt, odp... ?